internal/workflow.go

```diff
@@ -137,20 +186,16 @@ type (
 	// The interface is intended to simulate Go's select statement, and any Go select can be fairly trivially rewritten
 	// for a Selector with effectively identical behavior.
 	//
-	// For example, normal Go code like below:
+	// For example, normal Go code like below (which will receive values forever, until idle for an hour):
 	//  chA := make(chan int)
 	//  chB := make(chan int)
 	//  counter := 0
 	//  for {
 	//  	select {
-	//  	case i, more := <- chA:
-	//  		if more {
-	//  			counter += i
-	//  		}
-	//  	case i, more := <- chB:
-	//  		if more {
-	//  			counter += i
-	//  		}
+	//  	case x := <- chA:
+	//  		counter += i
+	//  	case y := <- chB:
+	//  		counter += i
```

Comment:
decided the ok/more/subtle difference in behavior wasn't worth addressing, as it made the code less normal and more complex.

for chans that are never closed and only contain the right values (no Receive errors), which is often the case in real code, both before and after behave identically.